README.md

```diff
@@ -16,41 +19,48 @@ For tutorials, samples, quick starts and other documentation, visit [Azure for J
 ### Prerequisites
 Java 8 or later is required to use the November 2019 client libraries, otherwise Java 7 or later is required.
 
-## Packages available
+## Release branches (Release tagging)
+
+### Master branch
+The master branch represent more recent code with new features and bug fixes. It does **not** represent latest released **GA** SDK. See [below](#Client-GA-November-2019-Releases) for latest **GA** release.<br/>
```

Comment:
Do you need `<br/>`? Markdown should be smart enough to provide breaks. To split a paragraph, add two line breaks.

Reply:
I tried without <br/> but I was not getting new line.